core/src/main/java/org/stellar/anchor/filter/AbstractJwtFilter.java

```diff
@@ -9,21 +10,19 @@
 import javax.servlet.http.HttpServletResponse;
 import lombok.SneakyThrows;
 import org.apache.http.HttpStatus;
-import org.stellar.anchor.api.exception.SepValidationException;
 import org.stellar.anchor.api.sep.SepExceptionResponse;
 import org.stellar.anchor.auth.JwtService;
-import org.stellar.anchor.auth.Sep10Jwt;
 import org.stellar.anchor.util.GsonUtils;
 import org.stellar.anchor.util.Log;
 
-public class JwtTokenFilter implements Filter {
+public abstract class AbstractJwtFilter implements Filter {
   public static final String JWT_TOKEN = "token";
-  private static final String OPTIONS = "OPTIONS";
-  public static final String APPLICATION_JSON_VALUE = "application/json";
-  private static final Gson gson = GsonUtils.builder().setPrettyPrinting().create();
-  private final JwtService jwtService;
+  static final String OPTIONS = "OPTIONS";
```

Comment:
Nit: does this need to be a variable? We are only using it in one place, it can be inlined instead.

Reply:
Removed.